test/tests/recognizePDFTest.js

```diff
@@ -1,4 +1,4 @@
-describe.skip("PDF Recognition", function() {
+describe("PDF Recognition", function() {
```

Comment:
Sounds good to me; I'm definitely in over my head here.

I'll rebase this in a second.